src/Morphir/Snowpark/JoinMapping.elm

```diff
@@ -84,16 +84,16 @@ processInternalJoin value mapValue ctx =
                 joinTypeName = getJoinType joinName
             in
             ([ Scala.ArgValue Nothing (Scala.Variable (Name.toCamelCase arg2))
```

Comment:
I think these spaces should be added to indicate that the items are part of the list `[...]` not the parenthesis expression `(...)`
That is:

```elm
    ([ 1
     , 3
     , 4
     , 5
     ]
    , 1
    )      
```

Instead of

```elm
    ([ 1
    , 3
    , 4
    , 5
    ], 1
    )
```

Or as **elm-format** does:

```elm
    ( [ 1
      , 3
      , 4
      , 5
      ]
    , 1
    )
```